packages/edit-post/src/components/layout/index.js

```diff
@@ -80,7 +80,8 @@ function Layout( {
 				aria-label={ __( 'Editor content' ) }
 				tabIndex="-1"
 			>
-				<EditorNotices />
+				<EditorNotices dismissible={ false } className="components-notice-list is-pinned" />
```

Comment:
We shouldn't be assigning `components-notice-list` here; and it seems like we shouldn't need to be either. If `components-notice-list` has its own class it wants to assign while allowing others to be included, the two should be merged from `NoticeList` itself. I'll see about making the change to your branch.